spec/functions/fqdn_rand_string_spec.rb

```diff
@@ -77,9 +51,7 @@ def fqdn_rand_string(max, args = {})
     charset = args[:charset]
     extra = args[:extra_identifier] || []
 
-    scope = PuppetlabsSpec::PuppetInternals.scope
-    scope.stubs(:[]).with("::fqdn").returns(host)
-    scope.stubs(:lookupvar).with("::fqdn").returns(host)
+    scope.stubs(:lookupvar).with("::fqdn", {}).returns(host)
```

Comment:
should this, for consistency, be handled with `let(:facts) {{ :fqdn => host }}`, or is that not possible here?